src/igvfd/schemas/in_vitro_system.json

```diff
@@ -0,0 +1,199 @@
+{
+    "title": "In Vitro System",
+    "$id": "/profiles/in_vitro_system.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "description": "Schema for submitting a sample grown in a in vitro system.",
+    "type": "object",
+    "required": [
+        "award",
+        "lab",
+        "source",
+        "donors",
```

Comment:
use singular donor not plural

Reply:
mixing still uses plural. Should I still changed it to singular?

Reply:
changed to singular in other biosample and concrete samples as well.